### PR TITLE
Fix undefined var in scaffolded molecule tests

### PR DESCRIPTION
**Pull request by asmacdo**

### CODE REVIEW

File: internal/plugins/ansible/v1/scaffolds/internal/templates/molecule/mkind/molecule.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ provisioner:
         samples_dir: ${MOLECULE_PROJECT_DIRECTORY}/config/samples
         project_dir: ${MOLECULE_PROJECT_DIRECTORY}
         operator_image: testing-operator
-        pull_policy: "Never"
+        operator_pull_policy: "Never"
```
Comment (camilamacedo86) [inline internal/plugins/ansible/v1/scaffolds/internal/templates/molecule/mkind/molecule.go:70]: it will be reverted? 
I think @fabianvf changed that.

Comment (fabianvf) [reply internal/plugins/ansible/v1/scaffolds/internal/templates/molecule/mkind/molecule.go:70]: It's a typo from me 😬, this is definitely a fix

**Comment by asmacdo** [comment]
@estroz I dont think the e2e tests need to change since they are passing.

**Comment by camilamacedo86** [comment]
PS.: @estroz @asmacdo 
The test are not checking it yet. See: https://github.com/operator-framework/operator-sdk/pull/3433

**Pull request merged**